assets/javascripts/discourse/components/whos-online.js.es6

```diff
@@ -1,36 +1,45 @@
-import Ember from "ember";
-var inject = Ember.inject;
+import Component from "@ember/component";
+import { inject as service } from "@ember/service";
+import { computed } from "@ember/object";
 
-export default Ember.Component.extend({
-  showWhosOnline: function () {
+export default Component.extend({
+  online: service("online-service"),
+
+  @computed
+  get showWhosOnline() {
```

Comment:
It should be either a computed property *or* a native getter, not both imo. The same with other properties here.